example.env.bt

```diff
@@ -1,5 +1,6 @@
 TRACING_STRATEGY=bt
 DOWNLOAD_PATH=pc
+DISPLAY_SELF_ASSESSMENT=true
```

Comment:
Should we also add these to the mobile-resources-repo?

Reply:
@zhaccc https://github.com/Path-Check/pathcheck-mobile-resources